source/Halibut.Tests/Support/LatestClientAndLatestServiceBuilder.cs

```diff
@@ -481,12 +484,16 @@ public async Task<ClientAndService> Build(CancellationToken cancellationToken)
                     service.Trust(serviceTrustsThumbprint);
                     listenPort = service.Listen();
                 }
-                else
+                else if(createTcpKillingDummyServiceWhenListeningService)
```

Comment:
By defaulting to a TCP killing service it was having unintended side effects for a test I was trying to adde.g. I really don't want anything to be there.

Reply:
I've left the default behavour so the builder provides a tcp killing service, but it might make sense for this to always be an opt-in. Future PR though!!